### DISCUSSION TITLE
Python-holidays transferred to vacanza organization

### DISCUSSION THREAD

**Discussion by dr-prodigy** [vacanza/holidays#1382] [Announcements]
Hi everyone reading!
This decision was taken some time ago, and it took time for me / us to be put in place, but we finally did it today..
python-holidays has finally stopped being my personal project, to move to a new organization (vacanza), of which I am of course part of, that from now on will be in charge of all its maintenance.

In fact, since I inherited this project (and I want to first of all say thank you to @ryanss to have given me this great opportunity, way back in 2018) this has almost never been a personal, but always a community one (as a good open source project should be) thanks to our numerous contributors (the list would be very long, you can read it in the project pages, but I want to surely mention @mborsetti as first.. thank you so much for the great work and long time we spent together, Mike! :-) ), casual or stable, to whom goes all my gratitude for all the good work done together.

This "last" step, anyway, has valid reasons to happen, especially now: since some months ago I could hand over the whole project maintenance (which used to be my main occupation here) to a fantastic group of stable contributors / collaborators, who are working very hard since then to improve the library every day, and have brought it to a completely different level of completeness and organization about functionalities, code quality, and lifecycle.
A level I would have never achieved on my own, with my (always more) limited time.. and that means you @arkid15r @KJhellico ! Thank you guys!

Now that this is done, I feel like the project has reached a new maturity, and has finally moved to its next stage of life, to become a public project from every point of view.
This is surely good for me, but even more for the world, in some sense, and makes me very happy :-)

Meantime, of course, I will still stay "in the loop" (most probably behind the curtains, as I used to do in the "recentish" past), and keep on collaborating on this great project, which still means a lot to me.

That's all for now.

I'd like to say *thank you*, again, to whom up to now has supported, contributed, or just appreciated / used python-holidays, and to whom will do it in the future together with us.

YOU GUYS ROCK! Thanks so much for making this happen! Everybody cheers! <3

**Comment by minchinweb** [comment]
Any meaning to the organization name (*vacanza*)? Please tell me it's a cool holiday somewhere...

**Reply by dr-prodigy** [reply]
:-) "Vacanza" means exactly holidays in Italian..  @arkid15r 's proposal I "blessed" at once! :+1:

**Comment by arkid15r** [comment]
@dr-prodigy thank you for making even more great things possible by transferring the repository ownership to an organization!

**Comment by mborsetti** [comment]
Hello @dr-prodigy Maurizio,

Great news about this transition (thanks!) and thanks for all the work ... and the recognition (I really didn't do _that_ much!).  Great news and great to have @arkid15r @KJhellico lead this.

Buone vacanze!
